simple_test.sh

```diff
@@ -1,48 +1,81 @@
 #!/bin/sh
```

Comment:
Let's assume that we have bash. Let's also not assume that we know the location of the interpreter. Using `sh` also breaks the script as you've written it. `sh` does not have the double bracket expressions or `let`.

Change this to `#!/usr/bin/env bash`

Reply:
Good point, I was being silly for not making that change. I will update this.